bin/do_username

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env do_username
+
+require 'do_username'
+
+max_size = ARGV[0].to_i
+
+if max_size > 0
+  puts DOUsername.generate(max_size)
+else
+  puts DOUsername.generate
+end
```

Comment:
```suggestion
max_size = begin
             Integer(ARGV[0])
           rescue ArgumentError
             ARGV[0]
           end

puts DOUsername.generate(max_size)
```
I think we should let the library itself handle validation (rebasing this PR against master should help with that), so within this executable we should cast strictly to an int or pass the given argument directly through